.gitignore

```diff
@@ -1,2 +1,3 @@
 *.swp
 build
+custom-values-watcher.yaml
```

Comment:
If this is local to your development workflow, an alternative could be [$GIT_DIR/info/exclude](http://git-scm.com/docs/gitignore).
Also, it looks like the file is created in the `resources` directory?
Also see my remark about importing from a temporary location so this might not be necessary.

Reply:
Do you still need this now that the build is from a temporary directory?